lcrs_embedded/system/cpu_load.py

```diff
@@ -0,0 +1,51 @@
+"""
+Generate a load on the CPU with one process per core.
+"""
+import logging
+from multiprocessing import Process
+import time
+
+logger = logging.getLogger(__name__)
+
+def timed_cpu_load(endTime):
+    """
+    Add some numbers for a while.
+    Could have been placed inside cpu_load below but there it
+    is called within a separate process and is then not picked
+    up in test coverage. Solved with a direct call in the test.
+    """
+    m = 1
+    running = True
+    while running:
+        m = (m + 1) % 100000
```

Comment:
Is this a coincidental calculation or is it because it has a known effect on CPU?